appinventor/blocklyeditor/src/replmgr.js

```diff
@@ -441,6 +441,40 @@ Blockly.ReplMgr.putYail = (function() {
             poll();
 
         },
+        'chunker' : (function() {
+            var seq = 0;
+            var gensym = function() {
+                seq += 1;
+                return 'Q' + seq;
+            };
+
+            var chunker = function(input) {
+                var length = input.length;
+                var chunklen = 8192;
+                if (length < chunklen) {
```

Comment:
If `length == chunklen`, then we will end up creating a whole lot of extra work below. I think we should make this `length <= chunklen`.